app/views/cases/offender_sar/_partial_case_flags.html.slim

```diff
@@ -0,0 +1,28 @@
+.grid-row
+  .column-full.data-requests
+    h2.request--heading.data-requests__title
```

Comment:
Can we remove the data-requests__title class?

Reply:
done